examples/admin_policy/example_policy/example_policy/skypilot_policy.py

```diff
@@ -0,0 +1,74 @@
+import copy
+import getpass
+
+import sky
+
+
+class TaskLabelPolicy(sky.AdminPolicy):
+    """Example policy: add label for task with the local user name."""
+
+    @classmethod
+    def validate_and_mutate(
+            cls, user_request: sky.UserRequest) -> sky.MutatedUserRequest:
+        """Add label for task with the local user name."""
+        local_user_name = getpass.getuser()
+
+        # Add label for task with the local user name
+        task = user_request.task
+        for r in task.resources:
+            r.labels['local_user'] = local_user_name
+
+        return sky.MutatedUserRequest(
+            task=task, skypilot_config=user_request.skypilot_config)
+
+
+class ConfigLabelPolicy(sky.AdminPolicy):
+    """Example policy: add label for skypilot_config with the local user name."""
+
+    @classmethod
+    def validate_and_mutate(
+            cls, user_request: sky.UserRequest) -> sky.MutatedUserRequest:
+        """Add label for skypilot_config with the local user name."""
+        local_user_name = getpass.getuser()
+
+        # Add label for skypilot_config with the local user name
```

Comment:
ditto

Also, this example dups the previous one in purpose? What's the intention?

Reply:
It is to show changing task level config vs global-level config. Do you think it confusing?

Reply:
To avoid being confusing, it should mutate another global config field, not labels.

Reply:
The main purpose of the task-level policy is to show how to change the task resources in an admin policy.
Just realized that setting labels in a task can cause failure due to #3965.
I just modified the task policy to enforce spot usage for all GPU tasks. Wdyt?

Reply:
SGTM